packages/expo-module-scripts/package.json

```diff
@@ -33,6 +33,7 @@
     "@expo/npm-proofread": "^1.0.1",
     "@testing-library/react-native": "^12.5.2",
     "@tsconfig/node18": "^18.2.2",
+    "@tsconfig/react-native": "^3.0.5",
```

Comment:
Just to confirm, is it intended to use the community-based [`@tsconfig/react-native`](https://www.npmjs.com/package/@tsconfig/react-native) instead of the [`@react-native/typescript-config`](https://www.npmjs.com/package/@react-native/typescript-config) from the React Native monorepo?

Reply:
We should use the official one, this was a mistake